OrbitCore/EventBuffer.cpp

```diff
@@ -12,42 +12,42 @@ using orbit_client_protos::CallstackEvent;
 
 EventTracer GEventTracer;
 
-std::vector<CallstackEvent> EventBuffer::GetCallstackEvents(uint64_t a_TimeBegin,
-                                                            uint64_t a_TimeEnd,
-                                                            ThreadID a_ThreadId /*= 0*/) {
-  std::vector<CallstackEvent> callstackEvents;
-  for (auto& pair : m_CallstackEvents) {
-    ThreadID threadID = pair.first;
+std::vector<CallstackEvent> EventBuffer::GetCallstackEvents(
+    uint64_t time_begin, uint64_t time_end, ThreadID thread_id /*= kAllThreadsFakeTid*/) {
+  std::vector<CallstackEvent> callstack_events;
+  for (auto& pair : callstack_events_) {
+    const ThreadID callstack_thread_id = pair.first;
     std::map<uint64_t, CallstackEvent>& callstacks = pair.second;
 
-    if (a_ThreadId == 0 || threadID == a_ThreadId) {
-      for (auto it = callstacks.lower_bound(a_TimeBegin); it != callstacks.end(); ++it) {
+    if (thread_id == SamplingProfiler::kAllThreadsFakeTid || callstack_thread_id == thread_id) {
```

Comment:
I think we should rename it to just kAllThreadsId, it is technically not real but this is implied anyways. (not for this PR)

Reply:
I'll do so! Thanks!